### ISSUE TITLE
Is this package active ?

### ISSUE THREAD

**Issue by o314** [JuliaData/AbstractTables.jl#4]
or does it need to be archived ?

There is such a tremendous amount of table interface in the julia data landscape,
We spend hours and hours trying to guess which one we should use

Can somebody clarify, thanks?

**Comment by ararslan** [comment]
No, this repository is not active. See [Tables](https://github.com/JuliaData/Tables.jl) and [DataFrames](https://github.com/JuliaData/DataFrames.jl).

**Comment by o314** [comment]
Thanks for the quick feedback

**Comment by nalimilan** [comment]
@quinnj Maybe remove the repo?
